Library/Homebrew/formulary.rb

```diff
@@ -132,7 +133,7 @@ def self.load_formula_from_api(name, flags:)
     namespace = "FormulaNamespaceAPI#{Digest::MD5.hexdigest(name)}"
 
     mod = Module.new
-    remove_const(namespace) if const_defined?(namespace)
+    remove_const(namespace.to_sym) if const_defined?(namespace)
```

Comment:
This was actually fine as a String, but Sorbet think (and Ruby docs also [imply](https://ruby-doc.org/core-2.6.2/Module.html#method-i-remove_const)) that it should be a Symbol.

Reply:
Maybe move the `to_sym` to the `namespace` variable instead.